crypto/keys/pem_signer_factory.go

```diff
@@ -67,3 +73,11 @@ func (f PEMSignerFactory) Generate(ctx context.Context, spec *keyspb.Specificati
 
 	return &keyspb.PrivateKey{Der: der}, nil
 }
+
+// SetPKCS11Module should be called before creating a PEMSignerFactory
```

Comment:
This comment is out-of-date ("should be called before creating a PEMSignerFactory"!)